emailCrawler.java

```diff
@@ -49,16 +41,14 @@ public static void main(String args[]) throws IOException {
 	public static void printEmail(ArrayList<String> urlList) throws IOException {
 		for (String url : urlList) {
 			Document doc = Jsoup.connect(url).get();
-
 			// email format
 			Matcher m = Pattern.compile("[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\\.[a-zA-Z0-9-.]+")
 					.matcher(doc.select("a").toString());
```

Comment:
Sorry, this program missed a sentence. I have declared the HashSet<String> emailList as a global variable, like this,
public class emailCrawler {
	// used to store email addresses which are grabbed and avoid duplicate.
	private static HashSet<String> emailList = new HashSet<>();

	public static void main(String args[]) throws IOException {
		String url = "http://" + args[0];